core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/IntelliJFactory.scala

```diff
@@ -103,4 +107,16 @@ object IntelliJFactory {
     val dependencyProvider = new DependencyProvider(intelliJDependencyProvider, pluginDependencyProvider)
     new IntelliJFactory(dependencyProvider, paths, driverConfig)
   }
+
+  def version(root: Path): IntelliJVersion = {
```

Comment:
logic duplicated in IntelliJProvider

Reply:
Replaced the implementation in `IntelliJProvider` with a call to this one.